Geometry/HGCalGeometry/src/HGCalMouseBite.cc

```diff
@@ -0,0 +1,78 @@
+#include "Geometry/HGCalGeometry/interface/HGCalMouseBite.h"
+#include "DataFormats/Math/interface/CMSUnits.h"
+
+//#define EDM_ML_DEBUG
+using namespace cms_units::operators;
+
+HGCalMouseBite::HGCalMouseBite(const HGCalDDDConstants& hgc, const bool rot) {
+  bool modeUV =
+      ((hgc.geomMode() == HGCalGeometryMode::Hexagon8) || (hgc.geomMode() == HGCalGeometryMode::Hexagon8Full));
+  if (modeUV) {
+    const std::vector<double> angle = {90.0, 30.0};
```

Comment:
Would be faster to define the angles as `90._deg, 30._deg` and then only use `convertRadToDeg()` in the debug section.